### PR TITLE
Uploading Chef push to pilot

### PR DESCRIPTION
**Pull request by uday-sg**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
*
*
*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@uday-sg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: chef-server-compliance-delivery-devops/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Chef Compliance DNS prefix"
+      }
+    },
+    "baseUrl": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/uday-sg/azure-quickstart-templates/add-chef-p2p/chef-server-compliance-delivery-devops",
```
Comment (singhkay) [inline chef-server-compliance-delivery-devops/azuredeploy.json]: Needs to be this repo location

File: chef-server-compliance-delivery-devops/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    "subnet4Name": "chef-nodes-workstation-subnet",
+    "subnet4Prefix": "10.7.4.0/24",
+    "subnet4ID": "[concat(variables('vnetID'),'/subnets/',variables('subnet4Name'))]",
+    "uniqueString": "[uniquestring(resourceGroup().id)]",
+    "deployment-api-version": "2015-01-01",
+    "compute-api-version": "2015-06-15",
+    "network-api-version": "2015-06-15",
+    "network-api-version2": "2016-03-30",
+    "storage-api-version": "2015-06-15",
+    "extension-api-version": "2015-05-01-preview",
```
Comment (singhkay) [inline chef-server-compliance-delivery-devops/azuredeploy.json]: extensions have the same api version as vms

File: chef-server-compliance-delivery-devops/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      "value": "GEN-PASSWORD"
+    },
+    "vm-size": {
+      "value": "Standard_D2"
+    },
+    "dnsLabelPrefix": {
+      "value": "GEN-UNIQUE"
+    },
+    "baseUrl": {
+      "value": "https://raw.githubusercontent.com/uday-sg/azure-quickstart-templates/add-chef-p2p/chef-server-compliance-delivery-devops"
```
Comment (singhkay) [inline chef-server-compliance-delivery-devops/azuredeploy.parameters.json]: locations needs to be this repo

File: chef-server-compliance-delivery-devops/azuredeploy.json (hunk 1 above)
Comment (uday-sg) [inline chef-server-compliance-delivery-devops/azuredeploy.json]: Is this

https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/chef-server-compliance-delivery-devops

File: chef-server-compliance-delivery-devops/azuredeploy.parameters.json (hunk 3 above)
Comment (uday-sg) [inline chef-server-compliance-delivery-devops/azuredeploy.parameters.json]: Is this
https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/chef-server-compliance-delivery-devops

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**